mapiproxy/servers/default/emsmdb/emsmdbp_object.c

```diff
@@ -3185,16 +3294,26 @@ static enum MAPISTATUS emsmdbp_object_sharing_metadata_property(struct emsmdbp_c
 						      sharing_properties, &retvals);
 	OPENCHANGE_RETVAL_IF(!data_pointers || retvals[0] != MAPI_E_SUCCESS, MAPI_E_NOT_FOUND, local_mem_ctx);
 
-	/* TODO: We do only support calendar invitation by now. Checks on PidLidSharingLocalType */
+	/* TODO: We do only support calendar by now. Checks on PidLidSharingLocalType */
 	if (strncmp((char *)data_pointers[0], "IPF.Appointment", strlen("IPF.Appointment")) != 0) {
 		talloc_free(local_mem_ctx);
 		return MAPI_E_NO_SUPPORT;
 	}
 
-	/* Check every retval is SUCCESS to build up the XML */
-	for (i = 1; i < SHARING_PROPS_COUNT; i++) {
+	/* Check every retval from shared properties is SUCCESS to build up the XML */
+	for (i = 1; i < 3; i++) {
```

Comment:
Can you come up with something more meaningful? `i < 3` is certainly correct, but I can't identify or detect any failure. Basically, I wouldn't be able to know which of the 3 properties have lead to this failure.